pkgs/build-support/cc-wrapper/ld-wrapper.sh

```diff
@@ -69,7 +73,7 @@ relocatable=
 # Find all -L... switches for rpath, and relocatable flags for build id.
 if [ "$NIX_DONT_SET_RPATH" != 1 ] || [ "$NIX_SET_BUILD_ID" = 1 ]; then
     prev=
-    for p in "${params[@]}" "${extra[@]}"; do
+    for p in "${extraBefore[@]}" "${params[@]}" "${extraAfter[@]}"; do
```

Comment:
I think not including `extraBefore` here was an oversight of whoever added it originally. Note that the debugging code below originally also skipped it.